redbot/cogs/general/general.py

```diff
@@ -410,7 +411,12 @@ def _bitsize(num: int):
                 value=_(
                     "\N{SPEECH BALLOON} Text: {text}\n"
                     "\N{SPEAKER WITH THREE SOUND WAVES} Voice: {voice}"
```

Comment:
```suggestion
                    "\N{SPEAKER WITH THREE SOUND WAVES} Voice: {voice}\n"
```
Missing a newline here, to match the current formatting